microsoft-365/admin/add-users/resend-user-password.md

```diff
@@ -22,7 +22,7 @@ ms.assetid: 9b838071-94aa-4b8e-8d84-d17ece1ee951
 description: "Learn how to reset password and send a email to a new Microsoft 365 user."
 ---
 
-# Resend a user's password - Admin Help
+# Resend a user's password - Admin help
```

Comment:
@lhcole613 - Same comment as above about product name and wondering why "- Admin help" is added.

Reply:
@SKjerland, same as above.

Reply:
@lhcole613 - Looks good!